diff --git a/node_modules/oh-vue-icons/icons/ai/index.js b/node_modules/oh-vue-icons/icons/ai/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/ai/index.js
rename to node_modules/oh-vue-icons/icons/ai/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/bi/index.js b/node_modules/oh-vue-icons/icons/bi/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/bi/index.js
rename to node_modules/oh-vue-icons/icons/bi/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/ci/index.js b/node_modules/oh-vue-icons/icons/ci/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/ci/index.js
rename to node_modules/oh-vue-icons/icons/ci/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/co/index.js b/node_modules/oh-vue-icons/icons/co/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/co/index.js
rename to node_modules/oh-vue-icons/icons/co/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/fa/index.js b/node_modules/oh-vue-icons/icons/fa/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/fa/index.js
rename to node_modules/oh-vue-icons/icons/fa/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/fc/index.js b/node_modules/oh-vue-icons/icons/fc/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/fc/index.js
rename to node_modules/oh-vue-icons/icons/fc/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/fi/index.js b/node_modules/oh-vue-icons/icons/fi/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/fi/index.js
rename to node_modules/oh-vue-icons/icons/fi/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/gi/index.js b/node_modules/oh-vue-icons/icons/gi/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/gi/index.js
rename to node_modules/oh-vue-icons/icons/gi/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/hi/index.js b/node_modules/oh-vue-icons/icons/hi/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/hi/index.js
rename to node_modules/oh-vue-icons/icons/hi/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/index.js b/node_modules/oh-vue-icons/icons/index.js
index 9281ab6..011333a 100644
--- a/node_modules/oh-vue-icons/icons/index.js
+++ b/node_modules/oh-vue-icons/icons/index.js
@@ -1,21 +1,21 @@
 // THIS FILE IS AUTO GENERATED
-export * from "./ai";
-export * from "./bi";
-export * from "./ci";
-export * from "./co";
-export * from "./fa";
-export * from "./fc";
-export * from "./fi";
-export * from "./gi";
-export * from "./hi";
-export * from "./io";
-export * from "./la";
-export * from "./md";
-export * from "./oi";
-export * from "./pi";
-export * from "./pr";
-export * from "./px";
-export * from "./ri";
-export * from "./si";
-export * from "./vi";
-export * from "./wi";
+export * from "./ai/index.mjs";
+export * from "./bi/index.mjs";
+export * from "./ci/index.mjs";
+export * from "./co/index.mjs";
+export * from "./fa/index.mjs";
+export * from "./fc/index.mjs";
+export * from "./fi/index.mjs";
+export * from "./gi/index.mjs";
+export * from "./hi/index.mjs";
+export * from "./io/index.mjs";
+export * from "./la/index.mjs";
+export * from "./md/index.mjs";
+export * from "./oi/index.mjs";
+export * from "./pi/index.mjs";
+export * from "./pr/index.mjs";
+export * from "./px/index.mjs";
+export * from "./ri/index.mjs";
+export * from "./si/index.mjs";
+export * from "./vi/index.mjs";
+export * from "./wi/index.mjs";
diff --git a/node_modules/oh-vue-icons/icons/io/index.js b/node_modules/oh-vue-icons/icons/io/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/io/index.js
rename to node_modules/oh-vue-icons/icons/io/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/la/index.js b/node_modules/oh-vue-icons/icons/la/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/la/index.js
rename to node_modules/oh-vue-icons/icons/la/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/md/index.js b/node_modules/oh-vue-icons/icons/md/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/md/index.js
rename to node_modules/oh-vue-icons/icons/md/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/oi/index.js b/node_modules/oh-vue-icons/icons/oi/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/oi/index.js
rename to node_modules/oh-vue-icons/icons/oi/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/pi/index.js b/node_modules/oh-vue-icons/icons/pi/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/pi/index.js
rename to node_modules/oh-vue-icons/icons/pi/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/pr/index.js b/node_modules/oh-vue-icons/icons/pr/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/pr/index.js
rename to node_modules/oh-vue-icons/icons/pr/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/px/index.js b/node_modules/oh-vue-icons/icons/px/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/px/index.js
rename to node_modules/oh-vue-icons/icons/px/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/ri/index.js b/node_modules/oh-vue-icons/icons/ri/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/ri/index.js
rename to node_modules/oh-vue-icons/icons/ri/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/si/index.js b/node_modules/oh-vue-icons/icons/si/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/si/index.js
rename to node_modules/oh-vue-icons/icons/si/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/vi/index.js b/node_modules/oh-vue-icons/icons/vi/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/vi/index.js
rename to node_modules/oh-vue-icons/icons/vi/index.mjs
diff --git a/node_modules/oh-vue-icons/icons/wi/index.js b/node_modules/oh-vue-icons/icons/wi/index.mjs
similarity index 100%
rename from node_modules/oh-vue-icons/icons/wi/index.js
rename to node_modules/oh-vue-icons/icons/wi/index.mjs
diff --git a/node_modules/oh-vue-icons/package.json b/node_modules/oh-vue-icons/package.json
index f11b4d3..67d1f3e 100644
--- a/node_modules/oh-vue-icons/package.json
+++ b/node_modules/oh-vue-icons/package.json
@@ -2,6 +2,21 @@
   "name": "oh-vue-icons",
   "version": "1.0.0-rc3",
   "description": "Include inline SVG icons from different popular icon packs in Vue 2 & 3 easily.",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.esm.min.js"
+      }
+    },
+    "./icons": {
+      "import": {
+        "types": "./types/icons.d.ts",
+        "default": "./icons/index.js"
+      }
+    }
+  },
   "keywords": [
     "icons",
     "vue",
